### DISCUSSION TITLE
@next/font how to load font by name?

### DISCUSSION THREAD

**Discussion by Phoenix-Alpha** [vercel/next.js#42754] [Help]
I am working on site builder where users can select fonts they will use in their sites. So what fonts the page will have can be seen on server side when the page is being built as they are retrieved from database. Does anyone know if it's possible to use @next/font in my case?

**Comment by jsefiani** [comment]
I actually have the same question, so far I manually imported the fonts and added them to an array of objects, which is then consumed by a dropdown. I'm just wondering if there is a solution that requires less work.

**Comment by horcrux4** [comment]
Same question - any solution @Phoenix-Alpha?
